### DISCUSSION TITLE
Cannot pass rankdir=LR to dot layout

### DISCUSSION THREAD

**Discussion by worthy7** [hpcc-systems/hpcc-js-wasm#102] [Q&A] [answered]
My output is exactly the same for -y. Are you sure it's working? (other options are just default, dot and svg)

**Comment by GordonSmith** [comment]
From the docs:  "By default, the coordinate system used in generic output formats, such as [attributed dot](https://graphviz.org/docs/outputs/canon/), [extended dot](https://graphviz.org/docs/outputs/canon/), [plain](https://graphviz.org/docs/outputs/plain/) and [plain-ext](https://graphviz.org/docs/outputs/plain/), is the standard cartesian system with the origin in the lower left corner, and with increasing y coordinates as points move from bottom to top. If the -y flag is used, the coordinate system is inverted, so that increasing values of y correspond to movement from top to bottom."

My understanding is that it will only work for the mentioned output formats, so change your output to one of the above to test?

**Comment by worthy7** [comment]
I cannot test that exactly, but it seems I misunderstood. I do think my format is standard dot, but I am really not familiar with all the different formats. I realized this may be the wrong thing I am looking for.

My issue is that the produced graph is way too wide, so I wanted to have it display vertically long instead of horizontally. Similar to what this stackoverflow produced: https://stackoverflow.com/questions/18078318/graphviz-drawing-graph-vertically

Is this possible?

**Comment by GordonSmith** [comment]
Yea "-y" is only useful if your "output" format is one of the above (not your input format).

Make sure you have `rankdir="TB"` (Top to bottom) and not `rankdir="LR"` (Left to right)

To get an answer to your question I suggest you post over here:  https://forum.graphviz.org/ where the layout experts live!

If you want a live playground, you can edit the following here:  https://observablehq.com/@gordonsmith/graphviz-quick-test

**Comment by worthy7** [comment, marked as answer]
For anyone else looking, you can change the rankdir directly in the dot file like so:

https://stackoverflow.com/a/48808742/1079267
